queries.sql

```diff
@@ -62,3 +62,69 @@ SELECT MIN(weight_kg) AS min_weight, MAX(weight_kg) AS max_weight FROM animals;
 
 -- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
 SELECT CAST(AVG(escape_attempts) AS DECIMAL(5,2)) AS average_escape_attempts FROM animals WHERE extract(YEAR FROM date_of_birth) BETWEEN '1990' AND '2000';
+
+-- JOIN QUERIES
+-- What animals belong to Melody Pond?
+SELECT 
+an.name AS animal_name,
+ow.full_name AS owner_name 
+FROM animals an 
+INNER JOIN owners ow
+ON ow.id = an.owners_id 
+WHERE ow.full_name = 'Melody Pond';
+
+-- List of all animals that are pokemon (their type is Pokemon).
+SELECT 
+an.name AS animal_name, 
+sp.name AS species 
+FROM animals an 
+INNER JOIN species sp
+ON sp.id = an.species_id
+WHERE sp.name = 'Pokemon';
+
+-- List all owners and their animals, remember to include those that don't own any animal.
+SELECT
+ow.full_name as owner_name,
+an.name as animal_name
+FROM owners ow
+LEFT JOIN animals an
+ON ow.id = an.owners_id
+
+-- How many animals are there per species?
+SELECT 
+sp.name AS species_name,
+COUNT(an.name) AS animal_count
+FROM animals an 
+INNER JOIN species sp
+ON sp.id = an.species_id
+GROUP BY sp.name;
+
+-- List all Digimon owned by Jennifer Orwell.
+SELECT animals.name as animal_name
+FROM animals
+INNER JOIN owners
+ON animals.owner_id = owners.id
+INNER JOIN species
+ON animals.species_id = species.id
+WHERE owners.full_name = 'Jennifer Orwell'
+AND species.name = 'Digimon';
```

Comment:
- [ ] Great effort in updating your query as mentioned by the previous reviewer; however, there are still some typographical errors in your code. Please take a moment to review and ensure that your query is functioning correctly. 
![image](https://github.com/gilberthappi/Vet-clinic-database/assets/55840999/96414100-0cc9-4b8b-91ca-122652a64364)
- [ ] Additionally, as requested by the previous reviewer, after updating your query, please verify that it satisfies the query requirements and, if necessary, update the screenshot in the Pull Request description.